cmd/_integration-tests/cli/cli_test.go

```diff
@@ -218,12 +218,12 @@ func goBuild(name, outputPath, relCodePath string, errChan chan error) {
 	errChan <- nil
 }
 
-// runServerAndClient execs a TEST-server and TEST-client and puts a
+// runServerAndClient execs a test-server and test-client and puts a
 // runReference to their interaction on the runRefs channel
 func runServerAndClient(path string, port int, debugPort int) runReference {
 	// From within a folder with a truss `service`
 	// These are the paths to the compiled binaries
-	const relativeServerPath = "/bin/TEST-server"
+	const relativeServerPath = "/bin/test-server"
```

Comment:
Seeing these values called a relative paths, and seeing them start with a slash caused a moment of cognitive dissonance. But this isn't a big deal. :)